use/developer/database-schema/database-cassandra.md

```diff
@@ -0,0 +1,34 @@
+# Database Schema
+
+### 1. Introduction
+At UCI, we leverage Cassandra as our database solution due to its distributed nature and remarkable data access speed. Cassandra's ability to distribute data across nodes enables seamless parallel data access, enhancing both read and write performance. we maintain a comprehensive history of bot conversations using the following schema.
+
+### 2. Table Schema
+
+* Table Name - XMessage
+
+|    Column Name   |     Type           |    Description                                   |
+| ---------------- | ------------------- | ------------------------------------------------ |
+|       id         |        UUID         | Unique identifier for the record.                |
+|    userId        |        Text         | User identifier associated with the message.     |
+|     fromId       |        Text         | Identifier for the sender of the message.        |
+|     channel      |        Text         | Channel through which the message was sent.      |
+|    provider      |        Text         | Provider of the messaging service.               |
+|   timestamp      | LocalDateTime       | Date and time when the message was timestamped.  |
+| messageState     |        Text         | Current state or status of the message.          |
+|   xMessage       |        Text         | Content of the message.                          |
+|      app         |        Text         | App is the name of bot                           |
+|    auxData       |        Text         |      |
+|   messageId      |        Text         | Unique identifier for fcm callback.               |
+|    replyId       |        Text         | We can store id which provied by your provider like gupshup, netcore etc. |
+|    causeId       |        Text         |          |
+|   sessionId      |        UUID         |                |
```

Comment:
Why are these empty?

Reply:
Don't know about the use case of these columns or why it was created. should I remove these columns from the documentation? @chinmoy12c